integ-test/src/test/java/org/opensearch/sql/legacy/JdbcTestIT.java

```diff
@@ -165,6 +165,7 @@ public void functionWithoutAliasShouldHaveEntireFunctionAsNameInSchema() {
     );
   }
 
+  @Ignore("Handled by v2 engine which returns 'name': 'substring(lastname, 1, 2)' instead")
```

Comment:
Do we need to keep this test?

Reply:
I checked our new IT and can find test for subquery. So this is covered.